docs/lib/analytics/configure.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Configure existing resources
```

Comment:
Should we be more explicit here and just say "Use existing AWS resources"

Reply:
That's better!